docs/using/event-types.md

```diff
@@ -57,6 +57,12 @@ curl -v -XPOST -H "Content-Type: application/json" http://localhost:8080/event-t
   "category": "business",
   "partition_strategy": "random",
   "enrichment_strategies": ["metadata_enrichment"],
+  "default_statistic": {
+    "messages_per_minute": 1000,	
+    "message_size":	5,
+    "read_parallelism":	10,
+    "write_parallelism": 5
```

Comment:
yeah, I think it makes sense to have `read_parallelism` and `write_parallelism` as 1 here. Because some people will just copy/paste this example without too much thinking.